solutions/cses-1191.mdx

```diff
@@ -0,0 +1,136 @@
+---
+id: cses-1191
+source: CSES
+title: Cyclic Array
+author: Chongtian Ma
+---
+
+## Explanation
+
+We can turn this into a graph problem, or more precisely, a tree. Let's add an edge from $u$ to $v$ if the maximum length of the subarray that starts at $v$ has to end at $u-1$, and so the next subarray must start at $u$. Remember the array is cyclic, so $u$ is not necessarily greater than $v$. This can be found using basic two pointers.
```

Comment:
> This can be found using basic two pointers.

elaborate on this

Reply:
on the implementation details? i dont think its necessary, its pretty basic? its pretty evident from the code

Reply:
well then why not just leave everything to the code to explain?
still best to give a sentence or two like
"This can be found by using two pointers, [what you do real quick]"

Reply:
ok then...